### DISCUSSION TITLE
My gradle build fails for no reason

### DISCUSSION THREAD

**Discussion by DRD2k8** [FabricMC/fabric-api#4110] [Mod Dev Support]
### Description

Whenever I build my fabric mod, the gradle build fails after 20 to 30 seconds and says "(Mod Directory)\build\loom-cache\remapped_working\loom_mappings_1_20_1_layered_hash_2198_v2.net.fabricmc.fabric-api-fabric-api-0.92.2+1.20.1.jar The process cannot access the file because it is being used by another process"! This even happens on architectury! Why does this happen!?

**Comment by rotgruengelb** [comment]
Maybe you have a Game Instance running or another Gradle process. Could also be an AV.

**Comment by CrypticVerse** [comment]
This happened to me the other day,  what I did was:

Make sure only the IDE (or terminal) is open, and if it still fails, goto task manager for windows and select all entries with "JDK platform binary" and end all those tasks.

Also, is your OS mac, linux, or windows?

**Comment by CrypticVerse** [comment]
> This happened to me the other day,  what I did was:
> 
> Make sure only the IDE (or terminal) is open, and if it still fails, goto task manager for windows and select all entries with "JDK platform binary" and end all those tasks.
> 
> Also, is your OS mac, linux, or windows?

 ~~Or, if on MacOS, close the task using activity monitor~~
Does not apply to MacOS or linux as said below.

**Comment by modmuss50** [comment]
This only happens on Windows, newer versions of loom will log out details about the process that is holding a lock on the file in question.
